index.rst

```diff
@@ -1,47 +1,159 @@
 :tocdepth: 1
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
-
-.. sectnum::
-
-
-.. Add content below. Do not include the document title.
-
-.. note::
-
-   **This technote is not yet published.**
 
 Introduction
 ============
 
-Say you have a repository named `meas_worst` and you want to change it to better reflect its usage as the best `pipe_tasks` package ever: `pipe_best`. Several other pacakges depend on `meas_worst`, and others may be currently working on `meas_worst` ticket branches. This document describes how to rename the repository with minimal disruption to the LSST build system and its users.
+Say you have a repository named ``meas_worst`` and you want to change it to better reflect its usage as the best ``pipe_tasks`` package ever: ``pipe_best``. Several other packages depend on ``meas_worst``, and other developers may be currently working on ``meas_worst`` ticket branches. This document describes how to rename the repository with minimal disruption to the LSST build system and its users.
 
 Before you get to this point, be sure you've created an RFC on the topic, gotten agreement about the package rename, and created an implementation ticket in which to do the work.
 
- NOTE: The document assumes you are part of the `lsst-dm` github organization, but can apply to other groups as well by replacing `lsst-dm` with your organization in the text below.
+NOTE: This document assumes you are part of the `lsst-dm` github organization, but it applies to other groups as well, by replacing `lsst-dm` with your organization in the text below.
 
 The process
------------
+===========
 
 Here are the basic steps:
 
 1. Fork
 2. Rename package
 3. Create branches in old and new package
-4. Rename code directories and files in new pacakge
-5. Rename ups files
-6. Search and replace in source code and docs
-7. Update `lsstsw/etc/repos.yaml`
-8. Mark old package as deprecated in Readme and via ImportErrors in python
+4. Rename code directories and files (**commit**)
+5. Rename ups files (**commit**)
+6. Search and replace in source code and docs (**commit**)
+7. Update :file:`lsstsw/etc/repos.yaml`
+8. Deprecate old package
 9. Branch dependent packages
-10. Update depdendent ups tables and import/include statements
-11. Update `confluence package history`_
+10. Update dependent packages
+11. Test in Jenkins
+12. Update `confluence package history`_ page
+13. Get reviewed
+14. Post on Community and Slack
+
+We will go through them one at a time in detail below.
 
 .. _Confluence package history: https://confluence.lsstcorp.org/display/DM/DM+Stack+Package+History
 
+Fork
+----
+
+Fork ``meas_worst`` from the `lsst` organization into the `lsst-dm` organization via the github interface. When you click the "Fork" button, github should give you a "Where should we fork this repository?" dialog: choose `lsst-dm`. This is the repository we will work in from now on.
```

Comment:
it may be worth clarifying which steps are done on the github web page and which are done on the user's computer.